crates/cxx-qt-gen/src/syntax/foreignmod.rs

```diff
@@ -163,36 +163,36 @@ pub fn self_type_from_foreign_fn(signature: &Signature) -> Result<Receiver> {
         if !receiver.attrs.is_empty() {
             return Err(Error::new(
                 receiver.span(),
-                "Attributes on the `self:` receiver are not supported",
+                "Attributes on the `self:` receiver are not supported!",
             ));
         }
 
         if receiver.mutability.is_some() {
             return Err(Error::new(
                 receiver.span(),
-                "mut on self (i.e. `&mut self`) are not supported, use `self: Pin<&mut T>` instead",
+                "mut on self (i.e. `&mut self`) are not supported! use `self: Pin<&mut T>` instead",
```

Comment:
`use` -> `Use` due to new sentence now.

Same for the other errors in this file.